Python/pywarpx/picmi.py

```diff
@@ -1683,6 +1683,10 @@ def add_diagnostic(self):
             self.diagnostic = pywarpx.Diagnostics.Diagnostic(
                 self.name, _species_dict={}
             )
+            # By default, disable particle output and field output
+            # (This will be reset in FieldDiagnostic and ParticleDiagnostic)
+            self.diagnostic.fields_to_plot = 'none'
```

Comment:
This is a good idea, but unfortunately this class is also used by `ReducedDiagnostic`. These two parameters are not used by that class.

Reply:
OK, yes, that's true. Would it be fine with you if I add:
```
if not isinstance(self, ReducedDiagnostic):
```
before setting these two attributes?

Reply:
That's getting a bit kludgy. I think it would be better to set these in the two classes, with the appropriate checks of whether they had already been set. Note that also `LabFrameFieldDiagnostic` doesn't use `write_species`.